README.md

```diff
@@ -22,60 +22,86 @@ PhoneGap plugin for managing application assets with javascript asset maps. Incl
 
-- delete files specified by their asset id in Array like; [ "img/cards/card001.jpg", "img/cards/card002.jpg " .. ] <br />
+- delete files specified by their asset id in Array <br />
 - if an asset id uses a path format and matches a folder, then the folder content will be deleted; img/cards <br />
+- if an asset id cannot be deleted (i.e. file resides in read-only memory) fail will be called with an error message
 - the array CAN contain one asset id
 
-### getFileURI()
+**Example**
+```
+wizAssets.deleteFiles(["img/cards/card001.jpg", "img/cards/card002.jpg"], successCallback, failCallback);
+```
 
-**wizAssets.getFileURI(String assetId, Function success, Function fail);**
+### wizAssets.getFileURI(assetId, success, fail)
 
 - A success returns a local URL string like file://documents/settings/img/cards/card001.jpg <br />
-- example;
+- A failure returns an error message
 
+**Example**
 ```
 wizAssets.getFileURI("img/ui/logo.jpg", successCallback, failCallback);
 ```
 
-### getFileURIs()
-
-**wizAssets.getFileURIs(Function success, Function fail);**
+### wizAssets.getFileURIs(success, fail)
 
 - A success returns a hashmap of asset id matching its local URL such as
+- A failure returns an error message
 
 ```
 {
 
     "img/ui/loader.gif"  : "/sdcard/<appname>/img/ui/loading.gif", 
     "img/cards/card001.jpg" : "file://documents/settings/img/cards/card001.jpg" 
 
-} 
+}
 ```
+
+**Example**
+```
+wizAssets.getFileURIs(successCallback, failCallback);
+```
+
+### Error handling
+
+| Code | Constant                     | Description                                                                                            |
+|-----:|:-----------------------------|:-------------------------------------------------------------------------------------------------------|
+|    1 | `ARGS_MISSING_ERROR`         | Arguments are missing in your call to WizAssets' method                                                |
+|    2 | `INVALID_URL_ERROR`          | Specified URL to download is invald                                                                    |
+|    3 | `CONNECTIVITY_ERROR`         | Download could not be processed because network could not be accessed                                  |
+|    4 | `HTTP_REQUEST_ERROR`         | HTTP status of the request is not successful (not 2XX)                                                 |
+|    5 | `HTTP_REQUEST_CONTENT_ERROR` | Content received in HTTP request could not be read                                                     |
+|    6 | `DIRECTORY_CREATION_ERROR`   | Creation of the directory to save the asset failed                                                     |
+|    7 | `FILE_CREATION_ERROR`        | Saving the asset failed                                                                                |
+|    8 | `JSON_CREATION_ERROR`        | Your call to WizAssets' method failed and its error could not be processed internally                  |
```

Comment:
Hmm, I think from a JS point of view, getting that code as a string would be really useful. That's imho the more common paradigm in JS.